plugins/woocommerce-blocks/assets/js/blocks/mini-cart/frontend.ts

```diff
@@ -35,6 +31,14 @@ declare global {
 }
 
 window.addEventListener( 'load', () => {
+	// Update the totals immediately on load.
+	updateTotals( select( CART_STORE_KEY ).getCartData() );
```

Comment:
We do this on load, but due to the new caching mechanism it will not always result in a new API request.